pyramid_debugtoolbar_mongo/operation_tracker.py

```diff
@@ -47,13 +47,12 @@ def _get_stacktrace():
 # Wrap Cursor._refresh for getting queries
 @functools.wraps(_original_methods['insert'])
 def _insert(collection_self, doc_or_docs, manipulate=True,
-            safe=False, check_keys=True, **kwargs):
```

Comment:
interface change (though that is debatable because of kwargs) would require a bump in version, `0.3.0`

Reply:
I think it would be better to check if it isn't going to break the project for older/newer versions of debugtoolbar. That's why I put the kwargs and got the key from it with a default.
I think it isn't going to break, but thinking and being sure are very distinct things...

btw, I found a travis file. You are using it?